client/src/pages/sniffers/SniffersPage/UrlPage.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import { SnifferType } from "../../../stores/sniffersStores";
+import { InputAdornment, TextField, Tooltip } from "@mui/material";
+import {
+  AiOutlineBank,
+  AiOutlineCopy,
+  AiOutlineDatabase,
+} from "react-icons/ai";
+import List from "@mui/material/List";
+import ListItem from "@mui/material/ListItem";
+import ListItemText from "@mui/material/ListItemText";
+import ListItemAvatar from "@mui/material/ListItemAvatar";
+import Avatar from "@mui/material/Avatar";
+
+interface UrlPageProps {
+  sniffer: SnifferType;
+}
+
+const UrlPage: React.FC<UrlPageProps> = ({ sniffer }) => {
+  return (
+    <div className="flex flex-col w-full items-center space-y-4 justify-center ">
+      <List
+        className="items-center flex flex-col space-y-4  bg-primary "
+        sx={{
+          width: "100%",
+          maxWidth: "50%",
+          borderRadius: "20px",
+        }}
+      >
+        <div className="flex justify-start"></div>
+        <ListItem>
```

Comment:
The items are identical structure wise, let's extract to an outer component and re-use it.